content/en/tracing/trace_collection/automatic_instrumentation/single-step-apm/kubernetes.md

```diff
@@ -30,36 +30,109 @@ In a Kubernetes environment, use Single Step Instrumentation (SSI) for APM to in
 
 Follow these steps to enable Single Step Instrumentation across your entire cluster. This automatically sends traces from all applications written in supported languages.
 
-**Note:** To instrument only specific namespaces or pods, see [Advanced options](#advanced-options).
+**Note:** To instrument only specific namespaces or pods, seeworkload targeting in[Advanced options](#advanced-options).
 
 1. In Datadog, go to the [Install the Datadog Agent on Kubernetes][11] page.
 1. Follow the on-screen instructions to choose your installation method, select an API key, and set up the Operator or Helm repository.
 1. In the **Configure `datadog-agent.yaml`** section, go to **Additional configuration** > **Application Observability**, and turn on **APM Instrumentation**.
-   
+
    {{< img src="tracing/trace_collection/k8s-apm-instrumentation-toggle.jpg" alt="The configuration block for installing the Datadog Agent on Kubernetes through the Datadog app" style="width:100%;" >}}
 
 1. Deploy the Agent using the generated configuration file.
 1. Restart your applications.
 
 ## Configure Unified Service Tags
 
-Unified Service Tags (USTs) apply consistent tags across traces, metrics, and logs, making it easier to navigate and correlate your observability data. Learn how to [set USTs for Kubernetes services][5].
+Unified Service Tags (USTs) apply consistent tags across traces, metrics, and logs, making it easier to navigate and correlate your observability data.
+
+### Prerequisites
+
+To use the recommended ddTraceConfigs approach, ensure you have:
+
+| Component | Minimum Version | Notes |
+|-----------|-----------------|-------|
+| datadog-agent | 7.66+ | Required for valueFrom support |
+| datadog-operator | 1.16.0+ | 1.13.0+ works with agent version override |
+| datadog-helm-chart | 3.120.0+ | Added valueFrom support |
+
+### Recommended: Configure USTs with ddTraceConfigs
+
+With SSI, you can automatically extract UST values from pod labels and metadata without modifying individual deployments:
```

Comment:
Add a note to say that `app-name` can be whatever label exists in your application that you want to use as the service name. And you can use multiple labels this way as well.